app/views/courses/show.html.erb

```diff
@@ -0,0 +1,56 @@
+<% provide(:title, t(".title")) %>
+<section class="course_details_area p_120">
+  <div class="container">
+    <div class="main_title">
+      <h2><%= @course.name %></h2>
+    </div>
+    <div class="row course_details_inner">
+      <div class="col-lg-6">
+        <div class="c_details_img">
+          <%= image_tag @course.image, class: "img-fluid" , size: "555 x 259"%>
+        </div>
+        <ul class="nav nav-tabs" id="myTab" role="tablist">
+          <li class="nav-item">
+            <a class="nav-link" id="home-tab" data-toggle="tab" href="#home" role="tab" aria-controls="home"
+              aria-selected="true"><%= t".objective" %></a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" id="contact-tab" data-toggle="tab" href="#contact" role="tab" aria-controls="contact"
+              aria-selected="false"><%= t".subjects" %></a>
+          </li>
+        </ul>
+        <div class="tab-content" id="myTabContent">
+          <div class="tab-pane fade" id="home" role="tabpanel" aria-labelledby="home-tab">
+            <div class="objctive_text">
+              <p><%= @course.description %></p>
+            </div>
+          </div>
+
+          <div class="tab-pane fade" id="contact" role="tabpanel" aria-labelledby="contact-tab">
+            <div class="objctive_text">
+              <ul class="list">
+                <%= render partial: "subject", collection: @course.subjects, as: :subject %>
+              </ul>
+            </div>
+          </div>
+
+        </div>
+      </div>
+      <div class="col-lg-5">
+        <div class="c_details_list">
+          <ul class="list">
+            <li>
+                <a href="#"><%= t".trainer" %>
+                    <span>
+                      <%= render partial: "trainer", collection: @course.users, as: :user %>
+                    </span>
+                </a>
+            </li>
+            <li><a href="#"><%= t".members" %> <span><%= count_trainee_in @course %></span></a></li>
+            <li><a href="#"><%= t".schedule" %> <span><%= @course.start_date %> <%= t".to" %> <%= @course.end_date %></span></a></li>
```

Comment:
sử dụng format datetiem trong I18n nhé
`<%= @course.start_date %> <%= t".to" %> <%= @course.end_date %>`